app/models/notifications/possible_duplicate_found_notification.rb

```diff
@@ -0,0 +1,25 @@
+module Notifications
+  class PossibleDuplicateFoundNotification < ::NotificationType
+    def absolute_user_ids
+      return [created_by_user.id] if created_by_user.present?
+
+      []
```

Comment:
i'm thinking raise an exception in this case so we can investigate why this would be nil - wondering if we might hit a race condition if this is triggered before audit is created, any way to find out?

Reply:
agreed